api/src/codec-to-json.ts

```diff
@@ -0,0 +1,47 @@
+import { bytes, codec, context } from "@typeberry/block";
+import type { FastifyReply, FastifyRequest } from "fastify";
+import { createErrorResponse } from "./error";
+import { kinds } from "./kinds";
+
+type Req = { Params: { id: string }; Body: { codec: string } };
+type SuccessRes = { data: { json: string } };
+type Res = { Body: SuccessRes };
+
+export function codecToJson(req: FastifyRequest<Req>, reply: FastifyReply<Res>) {
+  const descriptorId = req.params.id;
+  const descriptor = kinds.find((x) => x.name === descriptorId)?.clazz;
+  if (!descriptor) {
+    reply.status(400).send(createErrorResponse("Wrong descriptor id", `Descriptor ${descriptorId} does not exist.`));
+    return;
+  }
+
+  const decoded = codec.Decoder.decodeObject(
+    descriptor.Codec,
+    bytes.BytesBlob.parseBlob(req.body.codec),
+    context.tinyChainSpec,
```

Comment:
Is there a way to have chain spec configurable?

Reply:
yeah but the question is how to do that. we can:
- add `chainSpecId` to path params
- add relevant values of chain spec to request payload 
unfortunately both solutions break compatibility with JAMcha so I decided to postpone this decision

Reply:
Ah, okay! If `JAMcha` has `tiny` everywhere too, let's keep it this way. CC @ggwpez